baybe/surrogates/gaussian_process/core.py

```diff
@@ -107,22 +105,19 @@ def from_preset(preset: GaussianProcessPreset) -> GaussianProcessSurrogate:
         """Create a Gaussian process surrogate from one of the defined presets."""
         return make_gp_from_preset(preset)
 
-    def to_botorch(self) -> Model:  # noqa: D102
```

Comment:
@Scienfitz: one critical bug detected and fixed, by removing this override. Please try to follow the logic and confirm 🙈 Should have been clear that this is wrong, but the override has been put in place prior to the latest changes.

Reply:
fix seems right to me, no overrides of `to_botorch` should be necessary now, right?

Reply:
yes, not having the override is pretty much a safe bet. I never was necessary, but we introduced it as shortcut (to avoid going through the adapter model) for the special case of GP. But that was at the time where scaling was not yet included at all. That's why it broke in this PR. Unfortunately, these bug are notoriously hard to catch since silent 😶